<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriceWidget</class>
 <widget class="QWidget" name="PriceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>31</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;</string>
  </property>
  <layout class="QHBoxLayout" name="priceWidgetLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto Mono&quot;;
font-size: 13px;
font-weight: 400;</string>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPrice">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto Mono&quot;;
font-size: 16px;
font-weight: 700;
padding-left: 14px;
padding-right: 10px;</string>
     </property>
     <property name="text">
      <string>$31.58</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="counterPrice" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>9</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9</width>
       <height>8</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-image: url(:/icons/triangle_red);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPercent">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto Mono&quot;;
font-size: 13px;
font-weight: 700;
color: #c53f4f;
padding-left: 6px;</string>
     </property>
     <property name="text">
      <string>4.6%</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto Mono&quot;;
font-size: 13px;
font-weight: 400;
padding-right: 13px;</string>
     </property>
     <property name="text">
      <string>Volume (24)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVolume">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;Roboto Mono&quot;;
font-size: 16px;
font-weight: 700;</string>
     </property>
     <property name="text">
      <string>$18146</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
